docs/howto/deployment/wsgi/gunicorn.txt

```diff
@@ -21,16 +21,15 @@ Running Django in Gunicorn as a generic WSGI application
 ========================================================
 
 When Gunicorn is installed, a ``gunicorn`` command is available which starts
-the Gunicorn server process. At its simplest, gunicorn just needs to be called
-with the location of a module containing a WSGI application object named
-`application`. So for a typical Django project, invoking gunicorn would look
-like::
+the Gunicorn server process. At its simplest, gunicorn is called with the
+location of a module containing a WSGI application object named `application`.
```

Comment:
"is called" is passive voice and doesn't read well here. Perhaps:

"The simplest invocation of gunicorn is to pass only the name of a module containing a WSGI application called `application`, which for a typical Django project would look like:"

Reply:
Much better, thank you.